receiver/postgresqlreceiver/testdata/obfuscate/inputQueryPlan.json

```diff
@@ -0,0 +1 @@
+[{"Plan":{"Node Type":"Seq Scan","Parallel Aware":false,"Async Capable":false,"Relation Name":"pg_authid","Alias":"pg_authid","Startup Cost":"123","Total Cost":"456","Plan Rows":"789","Plan Width":"1011","Filter":"( rolcanlogin AND ( rolname = CURRENT_USER ) )"}}]
```

Comment:
This doesn't seem a good test case cuz I didn't see any difference compared with `expectedQueryPlan.json`

Reply:
updated, although the new one also got minor differences. one `"Inner Unique": true` -> `"Inner Unique":"?"`